README.md

```diff
@@ -56,26 +56,52 @@ Follow these steps to configure a service account for the backend application:
 
 1. Go to your project settings page on the [Firebase
    console](https://console.firebase.google.com).
-1. Select the **Service accounts** tab.
-1. Click **Manage all service accounts**.
+1. Click the **Settings** gear next to 'Project Overview' and then **Project settings**.
+1. Select **Service accounts** and click the link **Manage service account permissions**.
 1. In the **IAM & admin** page click **Create service account**.
 1. In the dialog, create an account with the following parameters:
    * Enter *playchat-servlet* in the **Service account name** field.
    * Select **Project** > **Owner** in the **Role** menu.
-   * Select **Furnish a new private key**.
+   * Check **CREATE KEY**.
```

Comment:
Is **Create key** a checkbox? If so, use select instead. Per https://developers.google.com/style/word-list#check

Ignore otherwise.

Reply:
No, it's not a checkbox. It's a button you can click.